### PR TITLE
Opprop poly ontology

### PR DESCRIPTION
**Pull request by CharlesZ-Chen**

### CODE REVIEW

File: src/ontology/OntologyAnnotatedTypeFactory.java [hunk 1]
```diff
@@ -65,17 +65,26 @@ public OntologyQualifierHierarchy(MultiGraphFactory f, AnnotationMirror bottom)
...
+//                }
             tops.remove(ONTOLOGY);
             tops.add(ONTOLOGY_TOP);
             return tops;
         }
 
+        @Override
+        protected Set<AnnotationMirror>
+        findBottoms(Map<AnnotationMirror, Set<AnnotationMirror>> supertypes) {
+            Set<AnnotationMirror> newBottoms = super.findTops(supertypes);
```
Comment (wmdietl) [inline src/ontology/OntologyAnnotatedTypeFactory.java]: Why `super.findTops` and not `findBottoms`?

Comment (CharlesZ-Chen) [inline src/ontology/OntologyAnnotatedTypeFactory.java]: Sorry, wrong method name. Should be `findBottoms`.

Comment (wmdietl) [inline src/ontology/OntologyAnnotatedTypeFactory.java]: Same mistake in similar method further down.

Comment (CharlesZ-Chen) [inline src/ontology/OntologyAnnotatedTypeFactory.java]: Already changed it. Second PR is here: #2

**Pull request merged**